source/security/authentication/aws-iam.txt

```diff
@@ -153,23 +153,33 @@ The driver checks for your credentials in the following sources in order:
       .. literalinclude:: /code-snippets/authentication/aws-env-variable.js
          :language: javascript
 
-.. important:: Retrieval of AWS Credentials
-   
-   Starting in MongoDB version 4.11, when you install the optional
-   ``aws-sdk/credential-providers`` dependency, the driver uses the AWS SDK
-   to retrieve credentials from the environment. As a result, if you
-   have a shared AWS credentials file or config file, the driver will
-   use those credentials by default.
-   
-   You can override this behavior by performing one of the following
-   actions:
-   
-   - Set ``AWS_SHARED_CREDENTIALS_FILE`` variable in your shell to point
-     to your credentials file.
-   - Set the equivalent environment variable in your application to point
-     to your credentials file.
-   - Create an AWS profile for your MongoDB credentials and set the
-     ``AWS_PROFILE`` environment variable to that profile name.
+Specifying AWS Credentials
+--------------------------
+
+When you install the optional ``aws-sdk/credential-providers`` dependency, the driver
+uses the AWS SDK to retrieve credentials from the environment. If you have a shared AWS
+credentials file or config file, the driver uses those credentials by default.
+
+To manually specify the AWS credentials to retrieve, you can use the ``AWS_CREDENTIAL_PROVIDER``
+property to specify the credential provider. The following example passes a provider chain
```

Comment:
S: [nit] since you repeat "specify"
```suggestion
To manually specify the AWS credentials to retrieve, you can set the ``AWS_CREDENTIAL_PROVIDER``
property to the credential provider. The following example passes a provider chain
```